cwltool/job.py

```diff
@@ -386,8 +348,7 @@ def run(self, pull_image=True, rm_container=True,
             try:
                 env = cast(MutableMapping[Text, Text], os.environ)
                 if docker_req and kwargs.get("use_container"):
-                    img_id = str(docker.get_from_requirements(
-                        docker_req, True, pull_image))
+                    img_id = self.get_from_requirements(docker_req, True, pull_image)
```

Comment:
cwltool/job.py:351: error: Incompatible types in assignment (expression has type "unicode", variable has type "str")

Reply:
thanks! forgot that the return value was wrapped in string